demos/collection/multitable.php

```diff
@@ -5,7 +5,7 @@
 require_once __DIR__ . '/../atk-init.php';
 
 // Re-usable component implementing counter
-class Finder extends \atk4\ui\Columns
+$finder = new class() extends \atk4\ui\Columns
```

Comment:
Good idea, you can even wrap this inside get_class() :D (like get_class(new class ()...{})), otherwise the additional instance might be very confusing, addTo will always create a new instance

And name it more like `$finderClass`.

Reply:
addressed